apigw/src/shared/config.ts

```diff
@@ -150,6 +154,8 @@ export const eadConfig =
       }
     : undefined
 
+export const externalId = process.env.EXTERNAL_ID || 'espoo-ad'
```

Comment:
Is there a reason to use `||` instead of `??` here? Or maybe we could remove the default value completely and use `required()`?
Also, the name isn't very clear (see the comment in the evaka-infra pull request)